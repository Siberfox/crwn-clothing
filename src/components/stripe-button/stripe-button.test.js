import React from "react";
import { shallow } from "enzyme";
import { shallowToJson } from "enzyme-to-json";

import StripeCheckoutButton from "./stripe-button.component";

it("should render StripeCheckoutButton component", () => {
  const wrapper = shallow(<StripeCheckoutButton />);
  expect(shallowToJson(wrapper)).toMatchSnapshot();
});
