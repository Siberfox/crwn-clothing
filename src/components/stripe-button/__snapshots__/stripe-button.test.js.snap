// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render StripeCheckoutButton component 1`] = `
<ReactStripeCheckout
  ComponentClass="span"
  amount={NaN}
  billingAddress={true}
  className="StripeCheckout"
  description="Your total is $undefined"
  image="https://svgshare.com/i/CUz.svg"
  label="Pay Now"
  locale="auto"
  name="CRWN Clothing Ltd"
  panelLabel="Pay Now"
  reconfigureOnUpdate={false}
  shippingAddress={true}
  stripeKey="pk_test_51HEylSJwtLW4Y2M2eqnNZ6IzA0CjObC8HeVyAdKWIGi2QeyuufhcbAAhXJ4Ljf72Ssa5uVS4XtOptSoEkyKboNUs00f3iNjPAL"
  token={[Function]}
  triggerEvent="onClick"
/>
`;
